ts/core/MathDocument.ts

```diff
@@ -411,7 +411,7 @@ export interface MathDocument<N, T, D> {
   /**
    * Rerender the MathItems on the page
    *
-   * @param {number=} start    The state to start rerendering at
+   * @param {number} [start]    The state to start rerendering at
```

Comment:
Are the brackets right?  We don't do this for any other variables.  Should we?

Reply:
There's multiple ways to indicate optional parameters:
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns
Coming from ChromeVox I used the *Google closure style* with `=`. I am not sure why I changed it here, maybe because there is no default value.

Yes, we should probably come to uniform way. For now I remove the brackets until we decide on **our** style.